ui/service.go

```diff
@@ -18,6 +18,9 @@ import (
 
 const (
 	templateDir = "ui/web/template"
+	header      = templateDir + "/header.html"
```

Comment:
I would propose other approach - instead of defining 3 different strings (`header`, `footer` and `navbar`), create a slice of  strings: `tmplFiles = []string{"header.html", "footer.html", "navbar.html"}`.

Then in the function `ParseTemplate()` add your file to this string array, prefix each array member with a `templateDir`, and pass the whole array to a `ParseFiles()` function - it is a [variadic function](https://pkg.go.dev/html/template#ParseFiles) and can accept array of strings.